docs/section4/4-18.md

```diff
@@ -0,0 +1,31 @@
+# Airside System - Rule 4-18    
+**Schema Version:** 0.0.13  
+**Mandatory Rule:** True  
+**Rule ID:** 4-18    
+**Rule Description:** HVAC fans in the baseline design model shall remain on during unoccupied hours in systems primarily serving computer rooms in the B_RMR.  
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 Schedules exception #3 in the proposed column  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None
+
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem 
+**Applicability Checks:** 
+
+1. Applies to hvac systems primarily serving computer rooms.
+
+**Function Call:** 
+
+1. get_HVAC_systems_primarily_serving_comp_rooms()
+
+**Manual Check:** None  
+ 
+**Rule Logic:**  
+**Applicability Check 1:** 
+- Get list of hvac systems that primary serve computer rooms: `hvac_systems_primarily_serving_comp_rooms_list_b = get_HVAC_systems_primarily_serving_comp_rooms (B_RMR)`
+- For each hvac_b in hvac_systems_primarily_serving_comp_rooms_list_b: `for hvac_b in hvac_systems_primarily_serving_comp_rooms_list_b:`
+    - Get operation during unoccupied periods: `operation_during_unoccupied_b = hvac_b.fan_systems[0].operation_during_unoccupied`  
+    **Rule Assertion:**  
+    - Case 1: For each applicable hvac system, if fan system operation during unoccupied periods is continuous in the B_RMR: `if operation_during_unoccupied_b == “CONTINUOUS”: PASS`
+    - Case 2: Else, for each applicable hvac system system: `Else: UNDETERMINED and raise_message "THE HVAC SYSTEM SERVES ZONE(S) THAT INCLUDE COMPUTER ROOMS AND THEREFORE MAY WARRANT CONTINUOUS OPERATION DURING UNOCCUPIED HOURS PER SECTION G3.1-4 SCHEDULES EXCEPTION #3 IN THE PROPOSED COLUMN. VERIFY WITH A MANUAL CHECK." `
```

Comment:
I think the outcomes here should align with 4-6 - i.e., PASS outcome if (unoccupied operation == CONTINUOUS AND min. OA>0) OR (unoccupied operation ==cycling AND min. OA=0), FAIL otherwise. Aligning with 4-6 will ensure that baseline and proposed schedules follow the same approach.